### DISCUSSION TITLE
Character Generator Journal missing?

### DISCUSSION THREAD

**Discussion by geoidesic** [moo-man/WFRP4e-FoundryVTT#1479] [Q&A]
I've installed the WFRP system but there's no Character Generator Journal? Also other journals that are shown in your video – none of them are appearing for me.

This is what I was expecting to see:
![Screenshot 2023-03-17 at 14 42 10](https://user-images.githubusercontent.com/4637995/225937013-e5a78d41-a8a5-4c6c-abd0-4809394f0248.png)

This is what I actually see:
![Screenshot 2023-03-17 at 14 42 45](https://user-images.githubusercontent.com/4637995/225937162-153f9b47-9b17-43ea-89a3-ce36cfd9d6b4.png)

**Comment by moo-man** [comment]
Few things, 1. The core module isn't installed or initialized and 2. It will look different now that the journals have been reformatted to v10

**Reply by geoidesic** [reply]
Looking different is one thing... but the issue is I don't have access to ANY journals from the system. Most importantly, the Character Creator.

**Comment by moo-man** [comment]
As I mentioned, the core module needs to be installed and initialized, afterwards the character creator will work

**Reply by geoidesic** [reply]
Where is this core module of which you speak? Does it have a name by which I may find it? :)

**Reply by moo-man** [reply]
https://foundryvtt.com/packages/wfrp4e-core
